sematic/abstract_future.py

```diff
@@ -148,6 +200,14 @@ def props(self) -> FutureProperties:
         """
         return self._props
 
+    @property
+    def kwargs(self) -> Dict[str, Any]:
+        """The keyword arguments to pass into the Sematic func at execition time.
+
+        May contain Futures for argument values.
+        """
+        return self._props.kwargs
```

Comment:
What is the purpose of this? Is it to avoid having to update `future.kwargs` around the codebase? If that is the case, let's bite the bullet. This is an internal API so no risk of breaking user code.